api/src/main/java/io/druid/indexer/RunnerTaskState.java

```diff
@@ -25,5 +25,5 @@
   WAITING,
   PENDING,
   RUNNING,
-  NONE; // is used for a completed task
+  NONE // is used for a completed task
```

Comment:
(minor) `is **also** used for a completed task` ?

Reply:
It looks like, but not sure. I don't want to change it in this PR.